### ISSUE TITLE
[Merge] MONDO:0044715 'metopic ridging-ptosis-facial dysmorphism syndrome' into MONDO:0032836 'Weiss-kruszka syndrome'

### ISSUE THREAD

**Issue by galyea123** [monarch-initiative/mondo#7687]
The condition listed in MONDO: Weiss-Kruszka Syndrome  MONDO:0032836 has exact match
https://omim.org/entry/618619 with dbxref OMIM:618619.  It is not currently tagged as rare in MONDO but it is a rare disease.  
Please tag it as NORD rare  and add the Orphanet synonym: "Metopic ridging-ptosis-facial dysmorphism syndrome"

https://www.orpha.net/en/disease/detail/502430

Thank you!  Gioconda Alyea

**Comment by sagehrke** [comment]
Dear @galyea123, 
Thank you for reaching out to the Mondo Disease Ontology, part of the [Monarch Initiative](https://monarchinitiative.org/)!

We have received your ticket and will review it as soon as possible. 

If you have any additional information or updates relevant to your inquiry, please add them in a comment on this ticket. Your input is valuable in helping us understand and resolve the issue effectively.

Thank you for your patience and for your contributions to Mondo! 

Sincerely,
The Mondo Team

**Comment by twhetzel** [comment]
Hi @galyea123 - since there is now a pipeline in place to collect terms that should be in the NORD rare subset, may I ask that you update your file to tag Weiss-Kruszka Syndrome MONDO:0032836 as NORD rare? I checked the term in Mondo and do not currently see that it has this tag. In the meantime, I will add the Orphanet synonym: "Metopic ridging-ptosis-facial dysmorphism syndrome".

**Comment by galyea123** [comment]
Hi Trish!  NORD lists already this condition as "metopic ridging-ptosis-facial dysmorphism syndrome" MONDO:0044715 which MONDO also lists separately from Weiss-Kruszka Syndrome MONDO:0032836.

I think that the problem is that MONDO is not listing the respective synonyms.

I would suggest to add the synonym Weiss-Kruszka Syndrome to metopic ridging-ptosis-facial dysmorphism syndrome and 'delete' Weiss-Kruszka syndrome (or viceversa) as they are the same condition.  The term "ZNF462 Disorder" should also be added as this is included in GeneReviews.

In our current process everytime that MONDO adds a synonym for a condition we list the information is included in NORD when the cycle is completed.


Please let me know if this is what you need,

Thank you!


Please do not feel obligated to respond to my email in the evenings or on weekends and holidays.



 [cid:673cbd0d-f84a-4db3-9ee8-a2df1367c1e4]

Gioconda Alyea, FMD, IMG, MS (she/her)

Rare Disease and Health Equity Specialist, Medical Affairs

National Organization for Rare Disorders

p:

617-249-7300

a:

1900 Crown Colony Drive., Suite 310, Quincy, MA 02169

w:

rarediseases.org<https://goo.gl/CSh3hs>



________________________________

**Comment by twhetzel** [comment]
Hi Gioconda! 

Based on your statement: 
I would suggest to add the synonym Weiss-Kruszka Syndrome to metopic ridging-ptosis-facial dysmorphism syndrome and 'delete' Weiss-Kruszka syndrome (or viceversa) as they are the same condition.

Are you suggesting the two Mondo terms be merged? I did see in another ticket (https://github.com/monarch-initiative/mondo/issues/7641) the merge was requested, but was unclear why the merge did not happen as part of that ticket.

**Comment by galyea123** [comment]
Yes, the terms should be merged as they are the same condition. Thank you!



Please do not feel obligated to respond to my email in the evenings or on weekends and holidays.



 [cid:9f799f59-ad0f-4551-bb28-9627f1f10436]

Gioconda Alyea, FMD, IMG, MS (she/her)

Rare Disease and Health Equity Specialist, Medical Affairs

National Organization for Rare Disorders

p:

617-249-7300

a:

1900 Crown Colony Drive., Suite 310, Quincy, MA 02169

w:

rarediseases.org<https://goo.gl/CSh3hs>



________________________________
